### ISSUE TITLE
When installing the postgres_scanner plugin, duck db crashes and exits

### ISSUE THREAD

**Issue by liwashington** [duckdb/postgres_scanner#267]
### What happens?

The other day, duck db released version 1.1.0, but when using the postgres_scanner extension, the program automatically exited and did not work. As shown in the following figure:
![postgres_scanner](https://github.com/user-attachments/assets/02fe12f2-32fa-4b18-ae52-cb6fe3a5cffe)

I want to build and use postgres_scanner on my Windows host, but I cannot succeed. So, can it be build using msys2 on Windows? What should we do if possible? thank you!

### To Reproduce

Download duckdb cli from url:https://github.com/duckdb/duckdb/releases/download/v1.1.0/duckdb_cli-windows-amd64.zip,
After unzip it, run duckdb, then execute: INSTALL postgres;
Duckdb has no display. After a while, the program exits。

### OS:

windows10,windows11

### PostgreSQL Version:

none

### DuckDB Version:

v1.1.0

### DuckDB Client:

duckdb.exe

### Full Name:

liwashington

### Affiliation:

There were no errors displayed throughout the entire process

### Have you tried this on the latest `main` branch?

- [X] I agree

### Have you tried the steps to reproduce? Do they include all relevant data and configuration? Does the issue you report still appear there?

- [X] I agree

**Comment by Mytherin** [comment]
Thanks for the report! This is most likely related to your installed MSVC version - see https://github.com/duckdb/duckdb/issues/13848

**Comment by liwashington** [comment]
> Thanks for the report! This is most likely related to your installed MSVC version - see [duckdb/duckdb#13848](https://github.com/duckdb/duckdb/issues/13848)

Thank you !That's indeed the issue. But can you provide a method for compiling under Windows?

**Comment by carlopi** [comment]
Given failure mode I also would assume this to be related to the linked issue, I would expect given behaviour that the segfault would be there on any instruction, and not be specific to the `INSTALL postgres` query.

This would be properly fixed in the upcoming `v1.1.1`, or workarounds can be found at https://github.com/duckdb/duckdb/issues/13848#issuecomment-2352674000

As for how to build [loadable] duckdb extensions in general, and postgres_scanner in particular, can you raise this as a GitHub discussion, sharing the steps you tried / where it goes wrong? Or even better on https://discord.duckdb.org?

The bottom line is always following the steps performed by CI for producing the binaries.
